main.js

```diff
@@ -0,0 +1,52 @@
+class Todo {
+    constructor() {
+        this.items = []
+    }
+    addItem(title, priority) {
+        let item = {};
+        item.title = title;
+        item.priority = priority;
+        item.id = this.items.length;
+        this.items.push(item);
+        return item.id;
+    }
+    removeItem(id) {
+        let findItem = this.items.find((e) => {
```

Comment:
suggestion: you can use `findIndex`, your code will be shorter and call loop only once